### DISCUSSION TITLE
"Save image" should save as shown in the notebook, including background

### DISCUSSION THREAD

**Discussion by minsa110** [microsoft/vscode-jupyter#9962] [New Ideas]
Feedback from PyCon 2022

"The "save image" button saves my image with transparent background so I can barely see my axes and title."

In the notebook (and as expected to be saved):
![image](https://user-images.githubusercontent.com/10833267/167355676-6a94d783-99e9-4cb7-8b65-8e6bc1ada6d6.png)

When I open the saved file:
![image](https://user-images.githubusercontent.com/10833267/167355708-3e4bd1c6-b636-487a-96f2-df5399b6d754.png)
(If you look closely, you can see the axes.. It's just black and the background of my dark themed machine is very dark grey...)

**Comment by greazer** [comment]
@minsa110, we discussed this for a few minutes. Overall, we don't think it's safe to override the transparency of the plot image. We are plotting on top of a white rectangle. 

This could also be addressed if/when core supports copying images to clipboard.
